todoList/todoList.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>체리마루가 되자</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <h3>🍒 체리마루가 되자 🍒</h3>
+    </header>
+    <nav class="nav">
+      <button class="nav_todayOnly">오늘만 보기</button>
+      <button class="nav_tmrOnly">내일만 보기</button>
+      <button class="nav_all">함께 보기</button>
+    </nav>
+    <main class="main">
+      <section class="main_left">
+        <h2 class="main_title">오늘 할 일</h2>
+
+        <div class="main_input">
+          <input
+            class="main_inputbox"
+            type="text"
+            placeholder="todo를 입력해주세요"
+          />
+          <input class="main_input_btn" type="button" value="&#x2b;" />
+        </div>
+      </section>
+
+      <section class="main_right">
+        <h2 class="main_title">내일 할 일</h2>
+        <ul class="main_right_todo">
+          <li>
+            <h4>
+              🧺빨래하기🧺
+              <button class="main_right_delete">🗑</button>
+            </h4>
+          </li>
```

Comment:
PR보니까 여기서 한 번 어려움을 겪었군요! button 태그 잘 사용해 주었네요 😆 이렇게 작성한다면 구조가 눈에 잘 들어오고 텍스트나 버튼만 따로 속성을 설정해주기 좋을 것 같아요
```suggestion
          <li>
            <h4>🧺빨래하기🧺</h4>
            <button class="main_right_delete">🗑</button>
          </li>
```

Reply:
@choichoijin 오 확실히 button이 밖에 있는게 관리하기가 더 쉽겠네요! 곰 아 워 요 💙